src/plugins/intel_cpu/src/nodes/ngram.h

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <node.h>
+
+#include <memory>
+#include <string>
+#include <vector>
+
+namespace ov {
+namespace intel_cpu {
+namespace node {
+
+class Ngram : public Node {
+public:
+    Ngram(const std::shared_ptr<ngraph::Node>& op, const GraphContext::CPtr context);
```

Comment:
- Use `ov` instead of `ngraph` wherever possible.
- Use reference GraphContext::CPtr &context.

Reply:
Done, thanks